packages/webapp-libs/webapp-core/src/components/typography/typography.tsx

```diff
@@ -64,7 +64,7 @@ export const ParagraphBold = forwardRef<HTMLParagraphElement, ParagraphProps>(
       <p
         ref={ref}
         {...props}
-        className={cn(`font-semibold leading-7 ${firstChildMargin ? '[&:not(:first-child)]:mt-6' : null}`, className)}
+        className={cn(`font-semibold leading-7 ${firstChildMargin ? '[&:not(:first-child)]:mt-6' : ''}`, className)}
```

Comment:
I think we can simplify it 

`        className={cn(
          'leading-7',
          {
            '[&:not(:first-child)]:mt-6': firstChildMargin,
          },
          className
        )}`

Reply:
Yup, sorry, I forgot :D